doc/source/management/data_at_rest_encryption.rst

```diff
@@ -220,18 +192,47 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. _ps.data-at-rest-encryption.redo-log:
+
+Redo Log Encryption
+================================================================================
+
+:Availability: This feature is **Alpha** quality
+
+InnoDB redo log encryption is enabled by setting the variable
+``innodb_redo_log_encrypt``. This variable has two values: ``MASTER_KEY`` and
+``KEYRING_KEY``.
+
+``MASTER_KEY`` uses the InnoDB master key to encrypt unique keys for each log
+file in the redo log header. ``KEYRING_KEY`` uses the ``percona_redo`` versioned
+key from the keyring. When ``innodb_redo_log_encrypt`` is set to
+``KEYRING_KEY``, each new redo log file is encrypted with the latest
+``percona_redo`` key from the keyring.
+
+System variables
+--------------------------------------------------------------------------------
+
+.. variable:: innodb_redo_log_encrypt
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-redo-log-encrypt``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default: ``KEYRING_KEY``
+
```

Comment:
It seems to be `OFF` by default which isn't documented:
mysql> show variables like 'innodb_redo_log_encrypt';
+-------------------------+-------+
| Variable_name           | Value |
+-------------------------+-------+
| innodb_redo_log_encrypt | off   |
+-------------------------+-------+
1 row in set (0.01 sec)

Reply:
The default has been changed accordingly